File: 백준/Bronze/2588. 곱셈/곱셈.js
var fs = require('fs');
var input = fs.readFileSync('dev/stdin').toString().split('\n');

var A = parseInt(input[0]);
var B = parseInt(input[1]);

var first = A*(B%10);
var second = A*(((B%100)-(B%10))/10);
var third = A*((B-(B%100))/100);

console.log(first);
console.log(second);
console.log(third);
console.log(first + (second*10) + (third*100));

File: 백준/Silver/1629. 곱셈/곱셈.py
a,b,c = map(int,input().split())

def divide(x,y):
    if y == 1:
        return x % c
    half = divide(x, y//2)
    if y%2==0:
        return half*half%c
    else:
        return half*half*x%c
    
ans = divide(a,b)
print(ans)